specifications/Disruptor/Disruptor_SPMC.cfg

```diff
@@ -0,0 +1,16 @@
+SPECIFICATION
+    Spec
+
+CONSTANTS
+    Writers = { w }
+    Readers = { r1 r2 r3 }
+    MaxPublished = 10
+    Size = 4
+    NULL = NULL
+
+INVARIANTS
+    TypeOK
+    NoDataRaces
+
+PROPERTIES
+    Liveliness
```

Comment:
You probably want to add ```CHECK_DEADLOCK FALSE``` so that model checking completes.

Reply:
If you do this the CI also wants you to add `"ignore deadlock"` to the model features in the `manifest.json`.